website/content/docs/get-started/developer-qs.mdx

```diff
@@ -321,6 +321,20 @@ System.out.println("Secret written successfully.");
 
 </CodeBlockConfig>
 
+<CodeBlockConfig lineNumbers>
+
+```Curl
+curl \
+    --header "X-Vault-Token: dev-only-token" \
+    --header "Content-Type: application/json" \
+    --request POST \
+    --data '{"data": {"password": "hashi123"}}' \
```

Comment:
I believe in all the other examples the password we're using is Hashi123 (with a capital H) so we probably want to keep that consistent

Reply:
whoops. Thank you!